src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java

```diff
@@ -7,11 +7,8 @@
     @JsonProperty("zip_file_name")
     public final String zipFileName;
 
-    @JsonProperty("jurisdiction")
-    public final String jurisdiction;
-
-    @JsonProperty("po_box")
-    public final String poBox;
+    @JsonProperty("container")
+    public final String container;
```

Comment:
will notification microservice use this field?

Reply:
yes, container is added in https://github.com/hmcts/reform-scan-notification-service/pull/62